parallel-install/pkg/components/component.go

```diff
@@ -12,6 +12,20 @@ const StatusUninstalled = "Uninstalled"
 
 const logPrefix = "[components/component.go]"
 
+//ComponentInstallation interface defines contract for Component installation and uninstallation.
+type ComponentInstallation interface {
+	//InstallComponent installs a component.
+	//The function is blocking until the component is installed or an error (including Helm timeout) occurs.
+	//See the helm.HelmClient.InstallRelease documentation for how the context.Context is used for cancellation.
+	InstallComponent(context.Context) error
+
+	//UninstallComponent uninstalls a component.
+	//The function is blocking until the component is uninstalled or an error (including Helm timeout) occurs.
+	//See the helm.HelmClient.UninstallRelease documentation for how the context.Context is used for cancellation.
+	UnInstallComponent(context.Context) error
```

Comment:
Little thing - Un**i**nstall, not Un**I**nstall

Reply:
👍